src/ngraph/pass/memory_layout.hpp

```diff
@@ -68,7 +69,7 @@ class ngraph::pass::MemoryManager
         block_state m_state;
     };
 
-    MemoryManager(size_t alignment = 1);
+    MemoryManager(size_t alignment = 1, bool disable_reuse = false);
```

Comment:
I was wondering if we could maybe add a test case for `SIMPLE`. It doesn't seem to be used anywhere